config/routes_into_teaching.yml

```
@@ -57,7 +57,7 @@ routes:
       - question: unqualified_teacher
         answers: ["*"]
       - question: location
-        answers: ["*"]
+        answers: ["yes"]
```

Does this need to be capitalised @spencerldixon ?

```suggestion
        answers: ["Yes"]
```